src/bin/nydusd/main.rs

```diff
@@ -324,8 +324,8 @@ fn append_virtiofs_subcmd_options(app: App<'static, 'static>) -> App<'static, 's
 }
 
 fn append_services_subcmd_options(app: App<'static, 'static>) -> App<'static, 'static> {
-    let subcmd = SubCommand::with_name("daemon")
-        .about("Run as a global daemon hosting multiple blobcache/fscache/virtiofs services.")
+    let subcmd = SubCommand::with_name("singleton")
```

Comment:
Maybe should update together with snapshotter: https://github.com/containerd/nydus-snapshotter/blob/0f88fb6a4f190c01bab42aba2bd47aca784546c2/pkg/process/manager.go#L318